packages/core/crates/core-packet/src/interaction.rs

```diff
@@ -517,8 +517,15 @@ where
         let channel_id = channel.get_id();
         debug!("going to bump ticket index for channel id {channel_id}");
         let current_index = self.bump_ticket_index(&channel_id).await?;
+        let price_per_packet = self
+            .db
+            .read()
+            .await
+            .get_ticket_price()
+            .await
+            .unwrap_or(*DEFAULT_PRICE_PER_PACKET);
```

Comment:
Shouldn't emit `warn!` if this happens?

Reply:
Yes, can add that. But still its a legitimate case.

Reply:
Done.